lib/seattleflu/id3c/cli/command/etl/redcap_det_uw_retrospectives.py

```diff
@@ -523,32 +527,38 @@ def present(redcap_record: dict, test: str) -> Optional[bool]:
     """
     result = redcap_record[test]
 
-    if not result or result.startswith('Reorder requested, '):
+    standardized_result = standardize_whitespace(result.lower()) if result else None
+
+    if not standardized_result or standardized_result.startswith('reorder requested,'):
         return None
 
     test_result_map = {
-        'Negative'                          : False,
-        'None detected'                     : False,
-        'None detected.'                    : False,
-        'Not detected (qualifier value)'    : False,
-        'Detected'                          : True,
-        'Detected (qualifier value)'        : True,
-        'Positive'                          : True,
-        'Cancel, order changed'             : None,
-        'Canceled by practitioner'          : None,
-        'Duplicate request'                 : None,
-        'Inconclusive'                      : None, # XXX: Ingest this someday as present = null?
-        'Inconclusive.'                     : None, # XXX: Ingest this someday as present = null?
-        'Indeterminate'                     : None, # XXX: Ingest this someday as present = null?
-        'PENDING'                           : None,
-        'Test not applicable'               : None,
-        'Wrong test ordered by practitioner': None,
+        'negative'                              : False,
```

Comment:
Right, I think it's still going to be messy until they commit to a list of possible values for these fields.  Stripping punctuation will eliminate a couple more from the list. I'll make that change, but otherwise best to wait and see if there's any progress on getting more consistent data coming in.